views/extras.tmpl

```diff
@@ -93,6 +93,7 @@
 					<a class="menu-item" data-keynav-skip href="/proposals" title="Proposals">Proposals</a>
 					<a class="menu-item" data-keynav-skip href="/market" title="Market">Market</a>
 					<a class="menu-item" data-keynav-skip href="/attack-cost" title="Decred Attack Cost">Attack Cost</a>
+					<a class="menu-item" data-keynav-skip href="/visualblocks" title="Visual Blocks">Visual Blocks</a>
```

Comment:
I've hesitated on this because the drop down menu is already quite full and I don't want to add something else to scan through.  I feel like a link to "Visualization" would work on the /blocks page.